src/frontend/nuxt.config.js

```diff
@@ -62,8 +62,7 @@ export default {
     */
     axios: {
         baseURL: 'http://django:8000',
-        browserBaseURL: process.env.NODE_ENV === 'production' ? `https://${process.env.HEROKU_APP_NAME}.herokuapp.com` : 'http://localhost:8000'
-    },
+        browserBaseURL: process.env.NODE_ENV === 'production' ? `https://${process.env.DOMAIN}` || `https://${process.env.HEROKU_APP_NAME}.herokuapp.com` : 'http://localhost:8000'    },
```

Comment:
the `}` ended up at the end of this line? moving it down
```suggestion
        browserBaseURL: process.env.NODE_ENV === 'production' ? `https://${process.env.DOMAIN}` || `https://${process.env.HEROKU_APP_NAME}.herokuapp.com` : 'http://localhost:8000'    
    },
```